src/rqt_console/console.py

```diff
@@ -142,5 +141,8 @@ def trigger_configuration(self):
     def _subscribe(self, topic):
         if self._subscriber:
             self._context.node.destroy_subscription(self._subscriber)
-        self._subscriber = self._context.node.create_subscription(Log, topic, self.queue_message)
+        from rclpy.qos import qos_profile_system_default
```

Comment:
If there is no reason why this import is local please move it to the top of the file.

Reply:
A code reflex, to not import modules that cause C extension loading at the module level. It doesn't make sense really. Fixed in https://github.com/ros-visualization/rqt_console/pull/10/commits/ace4a4fb90cac203a2e0f31bf5ac17bbb20800a9.